motionplan/motionPlanner.go

```diff
@@ -38,71 +38,32 @@ type motionPlanner interface {
 
 type plannerConstructor func(frame.Frame, *rand.Rand, golog.Logger, *plannerOptions) (motionPlanner, error)
 
-// PlanMotion plans a motion to destination for a given frame. It takes a given frame system, wraps it with a SolvableFS, and solves.
-func PlanMotion(ctx context.Context,
-	logger golog.Logger,
-	dst *frame.PoseInFrame,
-	f frame.Frame,
-	seedMap map[string][]frame.Input,
-	fs frame.FrameSystem,
-	worldState *frame.WorldState,
-	constraintSpec *pb.Constraints,
-	planningOpts map[string]interface{},
-) ([]map[string][]frame.Input, error) {
-	return motionPlanInternal(ctx, logger, dst, f, seedMap, fs, worldState, constraintSpec, planningOpts)
-}
-
-// PlanFrameMotion plans a motion to destination for a given frame with no frame system. It will create a new FS just for the plan.
-// WorldState is not supported in the absence of a real frame system.
-func PlanFrameMotion(ctx context.Context,
-	logger golog.Logger,
-	dst spatialmath.Pose,
-	f frame.Frame,
-	seed []frame.Input,
-	constraintSpec *pb.Constraints,
-	planningOpts map[string]interface{},
-) ([][]frame.Input, error) {
-	// ephemerally create a framesystem containing just the frame for the solve
-	fs := frame.NewEmptyFrameSystem("")
-	if err := fs.AddFrame(f, fs.World()); err != nil {
-		return nil, err
-	}
-	destination := frame.NewPoseInFrame(frame.World, dst)
-	seedMap := map[string][]frame.Input{f.Name(): seed}
-	solutionMap, err := motionPlanInternal(ctx, logger, destination, f, seedMap, fs, nil, constraintSpec, planningOpts)
-	if err != nil {
-		return nil, err
-	}
-	return FrameStepsFromRobotPath(f.Name(), solutionMap)
+type PlanRequest struct {
```

Comment:
👏 thanks for making this a struct

Reply:
Are there any changes in this file other than integratingthe PlanRequest struct?